index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>JavaScript Basics - Challenge</title>
+  </head>
+  <body>
+    <script src="quiz.js"></script>
```

Comment:
I'm getting an error after inputing my answers on the prompt
<img width="538" alt="image" src="https://github.com/user-attachments/assets/fc1b300a-471e-49e1-8007-15802e95029e">

It seems `main` tag is missing, This can be fix by simply adding `<main />`
```suggestion
    <main />
    <script src="quiz.js"></script>
```